src/cfn/index.ts

```diff
@@ -1466,7 +1467,8 @@ export async function convertStackToIIDY(argv: Arguments): Promise<number> {
   const StackNameArg = parameterizeEnv(StackName).replace(/-\d+$/, '-{{build_number}}').replace(project, '{{project}}');
   const stackArgs: StackArgs = {
     Template: './cfn-template.yaml',
-    ApprovedTemplateLocation: 's3://template-location',
+    ApprovedTemplateLocation: 's3://approved-template-location',
+    PendingTemplateLocation: 's3://pending-template-location',
```

Comment:
By having a pending template location, users then can lock `ApprovedTemplateLocation` to a specific IAM group. Not sure if this goes in the right direction.